data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
+Credit Card     | Braintree  | 2.9% + $0.30 | At payday  | https://www.braintreepayments.com/pricing
 Paypal          |            | max(2%, $20) | At payday  |
-Bitcoin         |            | 1% + 15¢     | On request |
 
 # https://gratipay.com/about/faq#how-withdraw-funds
+
+
+#### Deserted
+
+What       | When                | Why
+-----------+---------------------+-------------------------
+Balanced   | 2015-06-11          | Service shutdown
+Bitcoin    | 2015-05-15 f20d7506 | Gratipocalypse
```

Comment:
So, Bitcoin is a pool. It just not tied to a Bank for a purpose of money transfer. So, we have **pools** (cash, bitcoin, account), **operators** (your pocket, bitcoin wallet/software, bank) and the routes. Routes are isolated **physically** (cash is parallel to bitcoin) so you need a **converter** (bank, exchange market, p2p service) to get from one pool **resource** to the other. Then routes can be **internal** (inside of bank, bitcoin to bitcoin), **local** (bank to bank in one country/region with same rules), **international** (two banks/regions with multiple rules).